### PR TITLE
Animation realism.

### PR DESCRIPTION
**Pull request by erd**

### CODE REVIEW

File: robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowAnimation.java [hunk 1]
```diff
@@ -1,165 +1,20 @@
...
-
-  /**
-   * Non-Android accessor.  Use to simulate end of animation.
-   */
-  public void invokeEnd() {
-    endAnimation();
-    new ShadowAnimationBridge(realAnimation).applyTransformation(1.0f, new Transformation());
-  }
-
   public void setLoadedFromResourceId(int loadedFromResourceId) {
```
Comment (jongerrish) [inline robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowAnimation.java]: Am I missing something, but this shadow now only has a single property with setter and gettter, but doesn't shadow any behaviour.... can't it be just deleted now?

File: robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowAnimator.java [hunk 2]
```diff
@@ -1,49 +1,8 @@
...
-import org.robolectric.annotation.RealObject;
 
-import java.util.ArrayList;
-import java.util.List;
-
-import static android.animation.Animator.AnimatorListener;
-
-@SuppressWarnings({"UnusedDeclaration"})
 @Implements(Animator.class)
 public class ShadowAnimator {
```
Comment (jongerrish) [inline robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowAnimator.java]: We could delete this too right?

File: robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowObjectAnimator.java [hunk 3]
```diff
@@ -1,195 +1,8 @@
...
-import org.robolectric.internal.Shadow;
-import org.robolectric.util.ReflectionHelpers.ClassParameter;
 
-import java.lang.reflect.Method;
-import java.util.ArrayList;
-import java.util.List;
-
-@SuppressWarnings({"UnusedDeclaration"})
 @Implements(ObjectAnimator.class)
 public class ShadowObjectAnimator extends ShadowValueAnimator {
```
Comment (jongerrish) [inline robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowObjectAnimator.java]: Can be deleted?

File: robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowAnimator.java (hunk 2 above)
Comment (erd) [inline robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowAnimator.java]: Yes.

File: robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowAnimation.java (hunk 1 above)
Comment (erd) [inline robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowAnimation.java]: Yes - I must have missed it after I rebased my changes. Feel free to remove it.

File: robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowObjectAnimator.java (hunk 3 above)
Comment (erd) [inline robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowObjectAnimator.java]: Yes.

**Pull request merged**